src/style.css

```diff
@@ -136,48 +133,63 @@ footer {
   gap: 1rem;
   z-index: 1;
   align-items: center;
-  background: rgb(63, 94, 251);
-  background:
-    radial-gradient(
-      circle,
-      rgba(63, 94, 251, 1) 0%,
-      rgba(252, 70, 107, 1) 100%
-    );
+  background: rgba(128, 125, 125, 0.966);
   padding: 2rem;
   width: 90vw;
-  left: 5%;
-  top: 5%;
+  top: 2%;
+  left: 2%;
+  right: 2%;
+  bottom: 2%;
+  margin: 2%;
   height: 90vh;
   overflow-y: auto;
+  border: 2px solid black;
+  border-radius: 4px;
+  box-shadow: 0 0 5px 4px rgb(75, 75, 75);
 }
 
-.cross {
-  width: 3rem;
-  position: absolute;
-  right: 0;
-  margin-right: 2rem;
-  cursor: pointer;
+.poUpCountainer p {
+  font-family: 'Roboto Mono', sans-serif;
+}
+
+.poUpCountainer p,
+.poUpCountainer strong {
+  font-size: 1.2rem;
+}
+
+.poUpCountainer::-webkit-scrollbar {
+  width: 8px;
+  border-radius: 3px;
 }
 
-.cross:hover {
-  background-color: aliceblue;
-  box-shadow: 5px 5px 5px rgba(140, 137, 134, 0.462);
+.poUpCountainer::-webkit-scrollbar-track {
+  -webkit-box-shadow: inset 0 0 6px rgba(0, 0, 0, 0.3);
 }
 
-.imgContainer {
-  width: 50%;
+.poUpCountainer::-webkit-scrollbar-thumb {
+  background-color: rgb(224, 215, 215);
+  border-radius: 4px;
+}
+
+.cross {
+  font-size: 2rem;
+  position: fixed;
+  top: 8%;
+  right: 4%;
+  margin-right: 2rem;
+  cursor: pointer;
+  color: white;
 }
 
 .commentImg {
-  width: 100%;
+  width: 50vw;
+  height: 30vw;
   border-radius: 3%;
-  transition-duration: 2s;
```

Comment:
It would be better if you use `transition-duration` property so that its scale will reduce smoothly.